.yamllint.yaml

```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  truthy: enable
```

Comment:
Do you only want to check for truthy? We could check for everything. Or do you want to enable the rules one-by-one?

Reply:
I think we will start with one rule. Since enabling everything at once will be overwhelming.